source/patterns/@aws-solutions-constructs/aws-fargate-sqs/test/fargate-sqs.test.ts

```diff
@@ -0,0 +1,379 @@
+/**
+ *  Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+ *  with the License. A copy of the License is located at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  or in the 'license' file accompanying this file. This file is distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES
+ *  OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions
+ *  and limitations under the License.
+ */
+
+import '@aws-cdk/assert/jest';
+import * as defaults from '@aws-solutions-constructs/core';
+import * as cdk from "@aws-cdk/core";
+import { FargateToSqs } from "../lib";
+import * as sqs from '@aws-cdk/aws-sqs';
+import * as ecs from '@aws-cdk/aws-ecs';
+
+test('New service/new queue, dlq, public API, new VPC', () => {
+
+  // An environment with region is required to enable logging on an ALB
+  const stack = new cdk.Stack(undefined, undefined, {
+    env: { account: "123456789012", region: 'us-east-1' },
+  });
+  const publicApi = true;
+  const clusterName = "custom-cluster-name";
+  const containerName = "custom-container-name";
+  const serviceName = "custom-service-name";
+  const queueName = "custom-queue-name";
+
+  new FargateToSqs(stack, 'test-construct', {
+    publicApi,
+    ecrRepositoryArn: defaults.fakeEcrRepoArn,
+    vpcProps: { cidr: '172.0.0.0/16' },
+    clusterProps: { clusterName },
+    containerDefinitionProps: { containerName },
+    fargateTaskDefinitionProps: { family: 'family-name' },
```

Comment:
oops - 'family-name' ought to be a constant like those defined in lines 27-31. This is an error in my code in aws-fargate-sqs.